core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -35,17 +42,23 @@ class ConsumerBounceTest extends BaseRequestTest with Logging {
   val topic = "topic"
   val part = 0
   val tp = new TopicPartition(topic, part)
+  val maxGroupSize = 5
 
   // Time to process commit and leave group requests in tests when brokers are available
   val gracefulCloseTimeMs = 1000
   val executor = Executors.newScheduledThreadPool(2)
 
   override def generateConfigs = {
+    generateKafkaConfigs()
+  }
+
+  private def generateKafkaConfigs(maxGroupSize: String = maxGroupSize.toString): Seq[KafkaConfig] = {
```

Comment:
Leaving this here for lack of a clear place. Integration tests are definitely nice to have, but I'd have expected we could hit more cases with a unit test in `GroupCoordinatorTest`.

Reply:
In response to this comment and https://github.com/apache/kafka/pull/6163#discussion_r252945186,

We discussed offline that it would be best to refactor these tests a bit as well. I have opened a follow-up JIRA to address that just so that this KIP can make it into 2.2
https://issues.apache.org/jira/browse/KAFKA-7893